test/karma.conf.js

```diff
@@ -38,15 +38,15 @@ module.exports = function(config) {
       'node_modules/es5-shim/es5-shim.js',
       'node_modules/sinon/pkg/sinon.js',
       'node_modules/video.js/dist/video.js',
-      'dist/videojs-contrib-ads.js',
-      'dist/videojs-contrib-ads.css',
-      'test/integration/lib/shared-module-hooks.js',
-      'test/dist/bundle.js',
+      {pattern: 'dist/videojs-contrib-ads.js', nocache: true},
+      {pattern: 'dist/videojs-contrib-ads.css', nocache: true},
+      {pattern: 'test/integration/lib/shared-module-hooks.js', nocache: true},
+      {pattern: 'test/dist/bundle.js', nocache: true},
```

Comment:
This allows you to refresh the Karma Chrome instance and get the latest built files.